tidy-viewer: add page, tv: convert to disambiguation (#22915)
tldr-pages/tldr: pages/common/tidy-viewer.md

# tidy-viewer

> A data pretty-printer with column styling for CSV, TSV, PSV, Parquet, and Arrow IPC files.
> Commonly aliased as `tv`.

> > More information: <https://github.com/alexhallam/tv>.

Reviewer: ```suggestion
> More information: <https://github.com/alexhallam/tv#help>.
```

> - Pretty-print a CSV file:

Reviewer: ```suggestion
- Pretty-print the first 25 rows of a CSV file:
```

`tidy-viewer {{path/to/file.csv}}`

- Pretty-print data piped from `stdin`:

`cat {{path/to/file.csv}} | tidy-viewer`

> - Pretty-print a file with a custom [s]eparator (e.g. a pipe-separated file):

Reviewer: ```suggestion
- Pretty-print a file with a custom [s]eparator (e.g. a pipe):
```

Reviewer 2: Done, thanks

`tidy-viewer {{[-s|--delimiter]}} '|' {{path/to/file.psv}}`

> - Display data with a specific [c]olor palette (1: nord, 2: one_dark, 3: gruvbox, 4: dracula, 5: solarized light):

Reviewer: This mnemonic is redundant.
https://github.com/tldr-pages/tldr/blob/e8f9d3e7ceb0b549f2383c1d7502171994069630/contributing-guides/style-guide.md#option-syntax
> If a command only supports short options or _**the short option greatly differs from the long option**_, attempt to document what the letter is short for with a mnemonic.
```suggestion
- Display data with a specific color palette (1: nord, 2: one_dark, 3: gruvbox, 4: dracula, 5: solarized light):
```

`tidy-viewer {{[-c|--color]}} {{3}} {{path/to/file.csv}}`

- Print all rows instead of the first 25, paging through the output:

> `tidy-viewer {{[-f|--force-all-rows]}} {{path/to/file.csv}} | less -S`

Reviewer: Color will be turned off by default if `tv` is writing to a pipe.
https://github.com/alexhallam/tv/blob/f71935e2e4b310a3f1c87634ea8f837f5e467ba0/README.md#titanic
> ```sh
> # send to pager with color
> # less 
> tv titanic.csv -ea | less -R
> ```
```suggestion
`tidy-viewer {{[-af|--color-always --force-all-rows]}} {{path/to/file.csv}} | less -RS`
```

Author: Applied. Thanks for the continuous support given, as always.
